src/app/containers/ConsentBanner/Banner/index.amp.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { arrayOf, bool, shape, string } from 'prop-types';
+import * as AmpHelpers from 'react-amphtml/helpers';
+import { ServiceContextConsumer } from '../../../contexts/ServiceContext';
+import { ConsentBanner } from '../../../components/ConsentBanner';
+import BannerText from './Text';
+
+const Accept = (message, onClick) => (
+  <AmpHelpers.Action events={onClick}>
+    {props => (
+      <button {...props} type="button">
+        {message}
+      </button>
+    )}
+  </AmpHelpers.Action>
+);
+
+const Reject = (message, href, onClick) => (
+  <AmpHelpers.Action events={onClick}>
+    {props => (
+      <a {...props} href={href}>
+        {message}
+      </a>
+    )}
+  </AmpHelpers.Action>
+);
+
+const AmpConsentBannerContainer = ({
+  type,
+  acceptAction,
+  rejectAction,
+  promptId,
+  hidden,
+}) => (
+  <ServiceContextConsumer>
+    {({ translations }) => {
+      const messaging = translations.consentBanner[type];
```

Comment:
```suggestion
      const consentConfig = translations.consentBanner[type];
```

For the same naming change reasons as the canonical in https://github.com/bbc/simorgh/pull/1428#discussion_r266863365

Reply:
![](https://media.giphy.com/media/mTtDdazMpaFi0/giphy.gif)